tests/src/test/java/org/eclipse/hono/tests/amqp/CommandAndControlAmqpIT.java

```diff
@@ -96,7 +101,7 @@
      * @param ctx The vert.x test context.
      */
     @BeforeEach
-    public void createRandomTenant(final VertxTestContext ctx) {
+    public void createRandomTenantAndDevice(final VertxTestContext ctx) {
```

Comment:
this method does not create a device, just a device ID

Reply:
I've renamed it now (`createRandomTenantAndInitDeviceId`).